File: examples/python/openclaw.py
# pyright: reportMissingImports=false

from chirpier import new_client, Log

client = new_client(api_key="chp_your_api_key")
client.log(
    Log(
        agent="openclaw.main",
        event="task.duration_ms",
        value=420,
        meta={"task_name": "email_triage"},
    )
)
client.flush()
client.close()

File: examples/go/openclaw.go
package main

import (
	"context"

	"github.com/chirpier/chirpier-go"
)

func main() {
	client, _ := chirpier.NewClient(chirpier.Options{Key: "chp_your_api_key"})
	defer client.Close(context.Background())

	_ = client.Log(context.Background(), chirpier.Log{
		Agent: "openclaw.main",
		Event: "tokens.used",
		Value: 1530,
	})

	_ = client.Flush(context.Background())
}
